### ISSUE TITLE
Missing decoupling capacitors

### ISSUE THREAD

**Issue by adailtonjn68** [jabadiagm/MSXgoauld_tn20k#14]
All the power pins should have decoupling capacitor. Their lack may cause instability.
I have seen this in many designs where the board used to restart itself. The issue is solve by using decoupling capacitors very close to the power pins.
It is nice to use them also in the voltage regulator.

I counted at least 15 places you should have decoupling capacitors.
A good value is 100 nF.

**Comment by jabadiagm** [comment]
You are right, revision 4 with capacitors on every chip and ground plane is currently under development.
Thank you for your comments

**Comment by adailtonjn68** [comment]
Perfect. 
Very nice work. Good luck

**Comment by jabadiagm** [comment]
All workng boards are including decoupling capacitors now
